server/api/router.go

```diff
@@ -124,6 +125,11 @@ func (s *serviceMiddlewareBuilder) middlewareFunc(next func(http.ResponseWriter,
 // @BasePath /pd/api/v1
 func createRouter(prefix string, svr *server.Server) *mux.Router {
 	rd := createIndentRender()
+	setAudit := func(labels ...string) createRouteOption {
```

Comment:
`RegistServiceForHTTP` is not only for audit I think.

Reply:
I will change `RegistServiceForHTTP ` name to "SetServiceAuditBackendForHTTP". Other actions such as rate limit may be processed by other `createRouteOption ` functions.